Formula/gcc.rb

```diff
@@ -33,6 +33,8 @@ def osmajor
     sha256 "679c9bfc2082f8ab4320c89082b08c4eab9523dd72bfed27fe4b712de7013a1f" => :mavericks
   end
 
+  # GCC's Go compiler is not currently supported on Mac OS X.
+  # See: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=46986
```

Comment:
@dunn it seems that go still won't work on OS X... the GCC configure script complains about unsupported language if you try to enable it